WireAPI/Sources/WireAPISupport/Mocks.swift

```diff
@@ -0,0 +1,79 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+/// This is just a placeholder to give the compiler some food.
+/// To auto generate mocks via 'sourcery' it uses the plugin configured in `Package.swift`.
+/// The generated mocks are processed from the sandbox directory and are not visible in the project folder:
+/// https://github.com/apple/swift-package-manager/blob/main/Documentation/Plugins.md#implementing-the-build-tool-plugin-script
+enum Mocks { }
+
+extension Mocks {
+
+    /// An Example how generated mocks are structured.
+    public class MockExample {
```

Comment:
I would just remove it.
Xcode shows you the signature of methods and properties.
Also, all this is defined in the stencil files.

Reply:
I get @caldrian point. The example was more for someone that onboard on the project.
I would add a topic section in Wire-iOS docC that mention we use Sourcery for mocks.

Reply:
Done